### ISSUE TITLE
stock assessment trend update on change of stock selection

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-tools-dev/online-advice#55]
_the selection needs to default to 'pick' the selection, it is not reasonable to expect a user to know that they should click on the stock each time they change the stock, unless you make it more obvious by having a specific click box to show the active selection_

**Cod selected**
![image](https://user-images.githubusercontent.com/23527005/173590624-f13035ec-c27c-4470-abe2-566a7fe8e706.png)

**Haddock charts**
![image](https://user-images.githubusercontent.com/23527005/173590703-3bf85d3c-717b-49fe-a0fa-774f46aaeabc.png)

**Comment by lucalamoni** [comment]
Hi @neil-ices-dk, could this solution using radio buttons work for the stock selection?
![Capture](https://user-images.githubusercontent.com/32708094/176875533-ecb432a2-5dc7-4524-ab63-d09fa9acbea3.PNG)

**Comment by neil-ices-dk** [comment]
@lucalamoni i think the radio buttons are clearer yes, you will need to have the first stock selected as default though on the first entry to this page

**Comment by lucalamoni** [comment]
@neil-ices-dk the app has been updated with the radio buttons to select a stock. There are still a couple of bugs:
- right now I managed to have only the last stock selected as a default, instead of the first
- the default selection is not really registered by the app server-side, so if you don't click anything and go straight to the assessment trends no stock is visualised by default.

Who could I ask in the data team to help me with some javascript to fix these bugs?
